### DISCUSSION TITLE
Cookbook authorship

### DISCUSSION THREAD

**Discussion by brian-rose** [ProjectPythia/projectpythia.github.io#258] [Ideas]
As the Cookbook initiative gains steam, I think we should put some thought into how to acknowledge individual content authors. I think this will be a critical part of building a strong community of contributors.

An example: if you look at the current [Radar Cookbook](https://projectpythiatutorials.github.io/radar-cookbook/landing-page.html), there is no mention of the fact that most of the content was authored by @mgrover1. 

Cookbooks will be "community owned" and GitHub will keep track of everyone who contributes any commits to each cookbook. But in my view, each Cookbook should state clearly and prominently who are the primary content author(s) -- essentially making a distinction between primary authors and everyone else who contributes edits, fixes, and updates.

That's obviously not a "hard" distinction in all cases, but should be clear enough most of the time so that we can build it into the [Cookbook template](https://github.com/ProjectPythiaTutorials/cookbook-template). The landing page of each Cookbook can declare primary authors right up front.

This is a little different than the attribution model we've used for Pythia Foundations, which is a broad collaboration among the core Pythia team. Cookbooks are by their nature going to be more personal and specialized.

**Comment by clyne** [comment]
Good points, and agreed that cookbooks are a little different and there is a lot of value in supporting individual author attribution. We should look to what has been done by the Earth Cube's [Peer Reviewed Notebooks. ](https://www.earthcube.org/notebooks) as an exemplar. A few things are worth noting:

1. Each of the "cards" for their [gallery](https://www.earthcube.org/notebooks) lists the authors (scroll to bottom)
2. Each card also has a "citation" button that takes you to a Zenodo entry
3. All of the READMEs have author attribution
4. Some of the rendered notebooks, such as this [one](https://nbviewer.org/github/earthcube2020/ec20_havlin_etal/blob/master/notebook/ec20_havlin_etal.ipynb),  look like a journal paper with title and author information up front.
